<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenNetlistView::NetlistTab</class>
 <widget class="QWidget" name="OpenNetlistView::NetlistTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalPathLayout">
     <item>
      <widget class="QLabel" name="constLabelPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modulehierarchy:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="OpenNetlistView::QNetListView" name="netlistView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenNetlistView::QNetListView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qnetlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
